php/Terminus/Models/Environment.php

```diff
@@ -707,6 +654,26 @@ public function sendCommandViaSsh($command)
         return $response;
     }
 
+    /**
+     * Formats environment object into an associative array for output
+     *
+     * @return array Associative array of data for output
+     */
+    public function serialize()
+    {
+        $info = [
+          'id' => $this->id,
+          'created' => date(Config::get('date_format', $this->get('created'))),
+          'domain' => $this->domain(),
+          'onserverdev' => $this->get('on_server_development') ? 'true' : 'false',
```

Comment:
I see... I suppose in this case it makes sense to use true/false as an output, it was just my JS eyes seeing things that way.